helsingborg-stad/app-mitt-helsingborg
# Refactor/appdelegate update

## Explain the changes you’ve made
Update ios-specific files according to react-native upgrade helper

## Explain why these changes are made
Our ios files differs from react-native own files for our current react-native versions. This can lead to harder times upgrading to newer react-native versions in the future.


## Explain your solution
Compared the project ios files against react-native upgrade helper and changed it accordingly.

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios`
3. The application should run as usual

## Tested environments

- [] Storybook on a iOS device/simulator.
